packages/api/src/command/medical/admin/redrive-dlq.ts

```diff
@@ -0,0 +1,93 @@
+import { SQS } from "aws-sdk";
+import { uniqBy } from "lodash";
+import BadRequestError from "../../../errors/bad-request";
+import { attributesToSend, getMessagesFromQueue, sqs } from "../../../external/aws/sqs";
+import { Config } from "../../../shared/config";
+import { uuidv4 } from "../../../shared/uuid-v7";
+
+const dlqUrl = Config.getSidechainFHIRConverterDQLURL();
+const regularUrl = Config.getSidechainFHIRConverterQueueURL();
+
+const maxNumberOfMessagesPerQuery = 10;
```

Comment:
OOC - why only 10 at a time?

Since the messages are small, shouldn't it be safe to pull larger amounts, say 1000?

Reply:
that's the maximum per query - [source](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html#API_ReceiveMessage_RequestParameters)